R/container.R

```diff
@@ -0,0 +1,41 @@
+#' Connect to an Azure Container
+#'
+#' @param tenant Character. The tenant ID.
+#' @param app_id Character. The app ID.
+#' @param ep_uri Character. The endpoint URI.
+#' @param container_name Character. The container name.
+#'
+#' @details All arguments default to environmental variables stored in your
+#'     .Renviron file. Note that you'll be routed automatically to the browser
+#'     for authentication if you don't have a cached token already.
+#'
+#' @return A blob_container/storage_container object.
+#'
+#' @export
+#'
+#' @examples
+#' \dontrun{get_container()}
+get_container <- function(
+    tenant = Sys.getenv("AZ_TENANT_ID"),
+    app_id = Sys.getenv("AZ_APP_ID"),
+    ep_uri = Sys.getenv("AZ_STORAGE_EP"),
+    container_name = Sys.getenv("AZ_STORAGE_CONTAINER")
+) {
+
+  token <- if (app_id != "") {
```

Comment:
wonder whether it's worth adding in something descriptive here around why this if statement exists... the arguments could posssibly do with tweaking? 

```suggestion
  # if the app_id variable is empty, we assume that this is running on an Azure VM,
  # and then we will use Managed Identities for authentication.
  token <- if (app_id != "") {
```